src/components/sectionList/listView/ManageListView.tsx

```diff
@@ -0,0 +1,110 @@
+import i18n from '@dhis2/d2-i18n'
+import { Button, Transfer } from '@dhis2/ui'
+import React, { useEffect, useMemo, useRef, useState } from 'react'
+import {
+    getColumnsForSection,
+    useModelSectionHandleOrThrow,
+} from '../../../lib'
+import css from './ManageListView.module.css'
+import { useModelListView, useMutateModelListViews } from './useModelListView'
+
+interface RenderProps {
+    handleSave: () => void
+    isSaving: boolean
+}
+type ManageColumnsDialogProps = {
+    onSaved: () => void
+    children: (props: RenderProps) => React.ReactNode
+}
+
+const toPath = (propertyDescriptor: { path: string }) => propertyDescriptor.path
+
+export const ManageListView = ({
```

Comment:
I think this will continue being the "wrapper" for the `view`. And then we can split into `ColumnsTransfer` and `FilterTransfer` if we needed when the time comes.